app/src/main/java/org/stepic/droid/notifications/LocalReminder.kt

```diff
@@ -13,4 +13,7 @@ interface LocalReminder {
     @MainThread
     fun userChangeStateOfNotification()
 
+    @MainThread
+    fun remindAboutRegistration()
```

Comment:
It is invoked from WorkerThread at https://github.com/StepicOrg/stepik-android/pull/192/files#diff-32fecc87d93c02d7c36b80e3604e7716R171. So, we should change annotation to worker thread and remove threadPool in internal implementation